ivy_tests/test_ivy/test_functional/test_core/test_general.py

```diff
@@ -133,7 +158,6 @@ def test_array_equal(
     backend_fw,
     fn_name,
     on_device,
-    ground_truth_backend,
```

Comment:
As `ground_truth_backend` is a required arg for `test_function` it has to be passed in here and also in the test_function call below. Running these tests throw an error.